packages/ses/package.json

```diff
@@ -91,6 +93,7 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-import": "^2.29.1",
+    "hermes-engine-cli": "^0.12.0",
```

Comment:
pin

Reply:
it's unlikely Hermes CLI will be updated [anytime soon](https://github.com/facebook/hermes/issues/1371#issuecomment-2052072367)

so could be worth pinning to avoid CI breaking on `master` in the near future

nb: we're running

https://github.com/leotm/endo/blob/0def87db44b43ea7fddb16fa4489a4176502b3f8/.github/workflows/ci.yml#L46

> If the `--immutable` option is set (defaults to true on CI), Yarn will abort with an error exit code if the lockfile was to be modified (other paths can be added using the [`immutablePatterns`](https://yarnpkg.com/configuration/yarnrc#immutablePatterns) configuration setting). For backward compatibility we offer an alias under the name of `--frozen-lockfile`, but it will be removed in a later release.